tests/test_svg_config.py

```
@@ -1,224 +1,48 @@
-            assert svg_config.db_data["host"] == "client.example.com" or svg_config.db_data["host"] == ""
+    def test_home_dir_sets_paths(self, temp_env, monkeypatch):
+        svg_config = reload_svg_config(monkeypatch)
+        assert svg_config.project == str(temp_env / "project")
+        assert svg_config.project_www == str(temp_env / "www")
+
+    def test_task_db_path_from_env(self, temp_env, monkeypatch):
+        monkeypatch.setenv("TASK_DB_PATH", str(temp_env / "tasks.db"))
+        svg_config = reload_svg_config(monkeypatch)
+        assert svg_config.TASK_DB_PATH == str(temp_env / "tasks.db")
+
+    def test_secret_key_from_env(self, temp_env, monkeypatch):
```

Ruff (0.14.1)

Unused method argument: `temp_env`

(ARG002)